src/servers/middlewares/auth-router.ts

```diff
@@ -114,4 +120,129 @@ authRouter.post(
   }),
 )
 
+authRouter.post(
+  "/login",
+  wrapAsyncToRunInSpan({
+    namespace: "auth-router",
+    fn: async (req: express.Request, res: express.Response) => {
+      try {
+        const ipString = isDev ? req?.ip : req?.headers["x-real-ip"]
+        const ip = parseIps(ipString)
+        if (ip === undefined) {
+          throw new Error("IP is not defined")
+        }
+
+        const code = req.body.authCode
+        const phone = checkedToPhoneNumber(req.body.phoneNumber)
+        if (phone instanceof Error) return res.status(400).send("invalid phone")
+
+        const loginResp = await Auth.loginWithPhoneCookie({
+          phone,
+          code,
+          ip,
+        })
+
+        if (loginResp instanceof Error) {
+          return res.status(500).send({ error: mapError(loginResp).message })
+        }
+
+        const cookies = setCookie.parse(loginResp.cookiesToSendBackToClient)
+        const csrfCookie = cookies?.find((c) => c.name.includes("csrf"))
+        const kratosSessionCookie = cookies?.find((c) =>
+          c.name.includes("ory_kratos_session"),
+        )
+        if (!csrfCookie || !kratosSessionCookie) {
+          return res
+            .status(500)
+            .send({ error: "Missing csrf or ory_kratos_session cookie" })
+        }
+
+        res.cookie(kratosSessionCookie.name, kratosSessionCookie.value, {
+          maxAge: kratosSessionCookie.maxAge,
+          sameSite: "none",
+          secure: kratosSessionCookie.secure,
+          httpOnly: kratosSessionCookie.httpOnly,
+          path: kratosSessionCookie.path,
+          expires: kratosSessionCookie.expires,
+        })
+
+        res.cookie(csrfCookie.name, csrfCookie.value, {
+          maxAge: csrfCookie.maxAge,
+          sameSite: "none",
+          secure: csrfCookie.secure,
+          httpOnly: csrfCookie.httpOnly,
+          path: csrfCookie.path,
+          expires: csrfCookie.expires,
+        })
+
+        res.send({
+          identity: {
+            id: loginResp.kratosUserId,
+            uid: loginResp.kratosUserId,
+            phoneNumber: phone,
+          },
+        })
+      } catch (e) {
+        addAttributesToCurrentSpan({ cookieLoginError: e })
+        return res.status(500).send({ result: "Error logging in" })
+      }
+    },
+  }),
+)
+
+// Logout flow
+// 1. Client app (web wallet/admin etc...) sends an HTTP GET
+//    to http://localhost:4002/auth/logout with credentials: 'include'
+// 2. The backend project logs the user out via kratos and
+//    revokes the session via kratosAdmin
+// 3. All the cookies are deleted via res.clearCookie
+//    from the backend
+// 4. The cookies should be magically deleted on the client
+authRouter.get(
+  "/logout",
+  wrapAsyncToRunInSpan({
+    namespace: "auth-router",
```

Comment:
same here. namespace `servers.middlewares.authRouter` and please add `fnName` with `logout`